ql/ql/src/codeql_ql/ast/Yaml.qll

```diff
@@ -64,4 +64,15 @@ class QlRefDocument extends YamlDocument {
       value = n.lookup("postprocess").(YamlSequence).getElement(_)
     )
   }
+
+  predicate isPrintAst() {
+    this.getFile().getStem() = "PrintAst"
+    or
+    exists(YamlMapping n, YamlScalar value |
+      n.getDocument() = this and
+      value.getValue().matches("%PrintAst%")
+    |
+      value = n.lookup("query")
+    )
```

Comment:
Maybe add a comment that this is for `.qlref` files.  
And maybe check that the file-extension is `.qlref`?

Reply:
Thanks for the review! That is included in the char pred and description of the class `QlRefDocument`. Is that acceptable?
However, I do need to add a comment for the predicate.

Reply:
Oh, I didn't notice that context.  

LGTM 👍